maven/docs/using-the-plugin.md

```diff
@@ -894,6 +894,8 @@ Generated properties
  |`git.commit.message.full`      | Represents the raw body (unwrapped subject and body) of the commit message (`git log -1 --pretty=format:%B`) |
  |`git.commit.message.short`     | Represents the subject of the commit message - may *not* be suitable for filenames (`git log -1 --pretty=format:%s`) |
  |`git.commit.time`              | Represents the (formatted) time stamp when the commit has been performed. |
+ |`git.commit.time.commit`       | Represents the (formatted) time stamp when the commit has been performed. |
```

Comment:
Could you perhaps also rename the remaining instances (including the ones in the tests) ? :-)

Sorry to be picky here.

Reply:
no worries what so ever :smile: ! I think i got them all now?